frontend/src/main/scala/bloop/engine/tasks/toolchains/ScalaJsToolchain.scala

```diff
@@ -121,13 +122,27 @@ object ScalaJsToolchain extends ToolchainCompanion[ScalaJsToolchain] {
   }
 
   override def artifactNameFrom(version: String): String = {
-    if (version.startsWith("0.6")) BuildInfo.jsBridge06
+    if (version.length == 3) sys.error("The full Scala.js version must be provided")
```

Comment:
I added this check because up until now it was sufficient to only specify 0.6 or 1.0 in the build configuration. Seed uses this convention, so old builds will be incompatible. The sbt plug-in is not affected, so I think we can avoid adding code to fall back to a specific Scala.js version.